monitoring/dashboard.json

```diff
@@ -16,35 +24,1214 @@
             "value": "zenko"
         },
         {
-            "name": "job",
-            "type": "constant",
-            "label": "job",
-            "description": "Name of the Cloudserver job, used to filter only the Cloudserver instances.",
-            "value": "artesca-data-connector-s3api-metrics"
-        }
-    ],
-    "editable": true,
-    "gnetId": null,
-    "graphTooltip": 0,
-    "id": 33,
-    "links": [],
-    "panels": [
+            "name": "job",
+            "type": "constant",
+            "label": "job",
+            "description": "Name of the Cloudserver job, used to filter only the Cloudserver instances.",
+            "value": "artesca-data-connector-s3api-metrics"
```

Comment:
Just out of curiosity, is it a good thing to hardcode the artesca prefix here? How is the testing of this feature within other CIs if the prefix is different?

Reply:
it is not really "hard-coded" : in production, the value is actually computed by Zenko-operator, and this value is actually not used.

However, having the value here makes it easier to load (manually) the dashboard in an artesca setup : grafana will prompt to get the values for the __inputs, but it will pre-populate with the values specified here...

Reply:
I see, thanks! :)